### PR TITLE
Small typo fixes

### PR DESCRIPTION
**Pull request by ali92hm**
First of all, great document. This is a great high level overview of what happens in k8s as commands are issued.

As I was reading this document in my text editor, I found a couple of typos that fixed along the way.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -151,7 +151,7 @@ That's a lot of steps! It's pretty amazing to follow the crumbs down the rabbit
 
 ## Initializers
 
-After an object is persisted to the datastore, it is not made fully visible by the apiserver or scheduled until a series of [intializers](https://kubernetes.io/docs/admin/extensible-admission-controllers/#initializers) have run. An initializer is a controller that is associated with a resource type and performs logic on the resource before it's made available to the outside world. If a resource type has zero initializers registered, this initialization step is skipped and resources are made visible immediately.
```
Comment (jamiehannaford) [inline README.md:154]: I believe keeping the plural `intializers` is correct

**Review by jamiehannaford — COMMENTED** [review]
This is great, thanks for the thoughtful PR. Just one piece of feedback.

File: README.md (hunk 1 above)
Comment (ali92hm) [reply README.md:154]: You're right I should make it plural, but it's still wrong in it's current form: `intializers` -> `initializers`

Comment (ali92hm) [reply README.md:154]: Fixed

**Comment by jamiehannaford** [comment]
🤘

**Pull request merged**